apps/oc_chef_wm/priv/dispatch.conf

```diff
@@ -131,6 +131,8 @@
 
 %% Cookbook artifact endpoints
 {["organizations", organization_id, "cookbook_artifacts", name, identifier], oc_chef_wm_named_cookbook_artifact_version, []}.
+{["organizations", organization_id, "cookbook_artifacts", name], oc_chef_wm_cookbook_artifacts, [{resource_args, single_artifact}]}.
```

Comment:
Guess I should have had a closer look at other examples, amended to respect the usual convention.